app/services/events/validate_creation_service.rb

```diff
@@ -62,7 +62,9 @@ def validate_create
       return invalid_code_error unless valid_code?
 
       invalid_persisted_event = persisted_event_validation(
-        customer.active_subscriptions.first || organization.subscriptions.find_by(id: params[:subscription_id]),
+        organization.subscriptions.find_by(
```

Comment:
this is strange by reading it, could we improve it?

Reply:
why strange?
`has_many :subscriptions, through: :customers`

Reply:
not the find_by but we use the result directly into a method call, would be better to put it into a var no?

Reply:
Agreed - Updated